js/app.js

```diff
@@ -0,0 +1,71 @@
+// Selectors for counters
+const TOTAL = 'total-zombies';
+const DEAD = 'dead-zombies';
+const SMALL = 'small-zombies';
+const MAD = 'mad-zombies';
+const STRONG = 'strong-zombies';
+
+let counters = {
+    [TOTAL]: 0,
+    [DEAD]: 0,
+    [SMALL]: 0,
+    [MAD]: 0,
+    [STRONG]: 0,
+};
+
+let calculateZombies = () => {
+    let totalCounter = 0;
+
+    zombiesData.map((zombieHP) => {
+        let zombieType = getZombieType(zombieHP);
+        let counter = counters[zombieType];
+
+        counters[zombieType] = ++counter;
+
+        totalCounter++;
+    });
+
+    counters[TOTAL] = totalCounter;
+};
+
+let getZombieType = (incomingHP) => {
+    let hp = parseInt(incomingHP);
```

Comment:
there are no requirements which are defines an integer value without decimal point
so, it's possible to get a zombie health value "10.5", "11.5" and so on

Reply:
fixed